resume-buddy/src/main/webapp/resume-review.html

```diff
@@ -6,24 +6,40 @@
     <link rel="stylesheet" href="resume-review-style.css">
     <script src="resume-review-script.js"></script>
   </head>
-  
+
   <body onload='getComments()'>
+
+    <div class="page-title">
+      <h1>Resume Review</h1>
+    </div>
+
+    <div id="adobe-dc-view"></div>
+    <script src="https://documentcloud.adobe.com/view-sdk/main.js"></script>
+    <script type="text/javascript">
+      document.addEventListener("adobe_dc_view_sdk.ready", function(){ 
+            var adobeDCView = new AdobeDC.View({clientId: "b98bbf69d44442479396583253ac267c", divId: "adobe-dc-view"});
+            adobeDCView.previewFile({
+              content:{location: {url: "http://www.africau.edu/images/default/sample.pdf"}},
+              metaData:{fileName: "sample.pdf"}
+            }, {});
+            });
```

Comment:
Nit: indentation should be 2 spaces for this section.

Reply:
Embedded PDF is not done. I just had this hardcoded to see how it would look. I'll change it will TODO

Reply:
If you want to, you can put the placeholder back in addition to the TODO. I was just happy to see it rendering a random PDF!